hack/wait-for-mcp.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
we need to check overlaps with https://github.com/openshift-kni/numaresources-operator/pull/44/commits/cb2d540cf0a87a63def2fcfcf6373ef5dbfe9258